app/translations/en/LC_MESSAGES/messages.pot

```diff
@@ -0,0 +1,27 @@
+# Translations template for PROJECT.
+# Copyright (C) 2024 ORGANIZATION
+# This file is distributed under the same license as the PROJECT project.
+# FIRST AUTHOR <EMAIL@ADDRESS>, 2024.
```

Comment:
Can we modify the babel.cfg so this information is correctly populated?

Reply:
No, not to my knowledge

Reply:
These can be set by passing in arguments to the `pybabel extract` command as documented here: [https://babel.pocoo.org/en/latest/cmdline.html](https://babel.pocoo.org/en/latest/cmdline.html)